integration-cli/docker_cli_daemon_test.go

```diff
@@ -2746,3 +2747,65 @@ func (s *DockerDaemonSuite) TestFailedPluginRemove(c *testing.T) {
 	// plugin should be gone since the config.json is gone
 	assert.ErrorContains(c, err, "")
 }
+
+func (s *DockerDaemonSuite) TestMigrateSnapshotter(c *testing.T) {
+	s.d.StartWithBusybox(testutil.GetContext(c), c)
+	defer s.d.Restart(c)
+
+	apiClient, err := client.NewClientWithOpts(client.FromEnv)
+	assert.NilError(c, err)
+	defer apiClient.Close()
+	info, err := apiClient.Info(testutil.GetContext(c))
+	assert.NilError(c, err)
+
+	driver := info.Driver
+	c.Log(info.Driver)
+
+	cli.Docker(
+		cli.Args("run", "-d", "--name", "top", "-p", "80", "busybox:latest", "top"),
+		cli.Daemon(s.d),
+	).Assert(c, icmd.Success)
+	/*
+			cli.Docker(
+				cli.Args("run", "-d", "--name", "top1", "-p", "1234:80", "--restart", "always", "busybox:latest", "top"),
```

Comment:
Is all this commented code intentionally present? :eyes:

Reply:
I thought I ripped all the `integration-cli` stuff out but some was lurking in another commit 😜